### DISCUSSION TITLE
View option: File instead of page

### DISCUSSION THREAD

**Discussion by jwildman16** [pdfarranger/pdfarranger#983] [Ideas]
I often combine multi-page PDF documents into a new single PDF. When adding multiple PDFs, there are a couple of frustrations: 

1. When adding multiple files at once (e.g., when dragging and dropping files), the files are not added in alphabetical order according to filename. Example: Adding "1 Intro.pdf" and "2 Summary.pdf" usually results in the pages of "2 Summary.pdf" being arranged ahead of "1 Intro.pdf." 
2. The application displays the files' individual pages, which is unwieldy when the combined document is often over 300 pages. I would prefer an interface similar to Acrobat's where I can drag and drop the files, they show up in alphabetical order, I can manually rearrange the entire file list, or I can expand individual files to manipulate individual pages. 

Since I rarely manipulate individual pages when combining PDFs, I'd much prefer the interface to just show the files by default. 

I'm not a programmer, so I can't contribute code, but I'd love to know whether this is a feature that will be considered. Thank you!

**Comment by kbengs** [comment]
Also requested here https://github.com/pdfarranger/pdfarranger/issues/1307

**Reply by mffm-NET** [reply]
Would be really useful. I also could imagine solving this in the View Window by sectionize the included Docs with a horizontal line to collapse and maybe moved as whole section.
Another solution idea were to order the multiple Files in column with vertical separating lines and its single pages as page per rows from top to bottom
But in general have a Preview of the first page of an added Document or "section" to rearrange like it's displayed in the Acrobat Pro window for merge PDFs. 
Another problem I discover often is when I like to double side print documents (merged to do all together) is to add empty pages between Documents to ensure the next one start with an odd page number (front side of paper also the right side if bounded)
